pages/en/zkapps/how-to-write-a-zkapp.mdx

```diff
@@ -396,6 +394,99 @@ Off-chain state is not yet available.
 
 </Alert>
 
+### Custom Tokens
+
+Custom tokens are a way to create new types of tokens that can be used in the zkApp ecosystem. Custom tokens are identified with a unique "token id" assigned to every kind of token when they are newly minted.
+
+Custom tokens are typically governed by zkApp accounts, where zkApp accounts can mint/burn tokens and facilitate transfers of two separate accounts. zkApps developers can use the supported SnarkyJS API for custom tokens to send, burn, and authorize transactions between other accounts.
+
+##### Custom Token API
+
+There are three main methods for zkApps to interact with custom tokens. This section will discuss how minting, burning, and sending custom tokens are exposed to zkApp developers.
+
```

Comment:
Suggestion: "This section shows you how to perform common token operations, such as minting, burning, and sending tokens, using SnarkyJS API."